### PR TITLE
4: avatar perf

### PR DESCRIPTION
**Pull request by chrisnojima**
This updates the avatar to use css and adds more caching (etc)
Master static production build: ~270ms
PR static production build: ~130ms

### CODE REVIEW

**Comment by chrisnojima** [comment]
- [ ] need https://github.com/keybase/client/pull/13848

**Comment by chrisnojima** [comment]
@keybase/react-hackers this makes avatars faster (cc: @cecileboucheron @adamjspooner )

**Comment by adamjspooner** [comment]
LGTM!

**Comment by cecileboucheron** [comment]
👍

File: shared/stories/shared-stories.js [hunk 1]
```diff
@@ -1,38 +1,38 @@
 // @flow
-import chat from '../chat/index.stories'
+// import chat from '../chat/index.stories'
```
Comment (MarcoPolo) [inline shared/stories/shared-stories.js]: I assume this isn't going to get merged?

**Pull request merged**